### PR TITLE
Add undo closed tab

### PR DESCRIPTION
**Pull request by nkarampi**
Added undo recently closed session.

Fixes: #44

### CODE REVIEW

**Comment by maciekmm** [comment]
Hi, I don't think a separate button is necessary. I'd like to have the _undo closed tab_ as an entry (option) in the container/tab context menu. Thanks for your contribution though :+1:

File: src/sidebar/native_context_menu.js [hunk 1]
```diff
@@ -32,6 +32,19 @@ function includesAny(arr, incl) {
...
+function restoreMostRecent(sessionInfos) {
+    if (!sessionInfos.length) {
+      console.log("No sessions found")
+      return;
+    }
+    let sessionInfo = sessionInfos[0];
+    if (sessionInfo.tab) {
+      browser.sessions.restore(sessionInfo.tab.sessionId);
+    } else {
+      browser.sessions.restore(sessionInfo.window.sessionId);
```
Comment (maciekmm) [inline src/sidebar/native_context_menu.js]: I don't think we should restore window session.

**Comment by nkarampi** [comment]
> Move button to menu entry. Don't restore window session.

I'm not sure what you mean by "menu entry" 

Would you provide a screenshot for me to understand the button placement?

**Comment by maciekmm** [comment]
@nkarampi 
the menu that pops up when you right-click a tab or a container.

**Comment by nkarampi** [comment]
@maciekmm Done.
Sorry I totally confused the 'native' menu, thus I made the mistake with the button.

**Comment by maciekmm** [comment]
Merged with some changes :+1: 
Just so You know: The ampersands in the message files are for the keyboard shortcuts, e.g "&Undo close tab" will make it possible to hit "U" when context menu is open to activate the menu item.

I actually removed the option from container context menu because it was confusing. There's currently no easy way to restore tabs of a specific container (as Tab object in getRecentSessions does not include cookieStoreId) so remembering last closed tabs would be needed in each container.

Thank You for your contribution, I appreciate it :+1:

**Pull request closed without merging**